packages/components/src/internal/components/administration/AccountSettingsPage.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { mountWithAppServerContext, waitForLifecycle } from '../../testHelpers';
+
+import { TEST_USER_GUEST, TEST_USER_READER } from '../../userFixtures';
+import { AdminAppContext } from '../../AppContext';
+import { InsufficientPermissionsPage } from '../permissions/InsufficientPermissionsPage';
+import { Page } from '../base/Page';
+import { UserDetailHeader } from '../user/UserDetailHeader';
+import { Notifications } from '../notifications/Notifications';
+
+import { AccountSettingsPage } from './AccountSettingsPage';
+
+beforeAll(() => {
```

Comment:
Whoa. I just realized we have two implementations of `getDateFormat()` and `getDateTimeFormat()`. One hands back the raw string from the context and the other hands back the moment format. The latter also already accepts an optional `Container`. We should rectify this by making the names of these methods distinct otherwise it will lead to misuse / misunderstanding.